lab: Fix inconsistencies in chapter introductions (#189)
open-education-hub/operating-systems: content/misc/lab-setup.md

# Setting up the Lab environment

If you have already cloned the repository, make sure it is updated:

```console
cd operating-systems
git pull --rebase
```

The command may fail if you have uncommitted changes.

> If that is the case, commit changes and retry.

Reviewer: Commit or stash and pop after `git pull`? I generally support commits, but last semester's experience showed me that maybe the simpler option is the best / fastest. @Alex-deVis, what do you say?

Reviewer 2: Right, but only say `git stash` -> `git pull` -> `git stash pop`. Give one comprehensive option, not a bunch.

Reviewer: This will confuse people. I wouldn't give options. One way to rule them all.

Reviewer 2: To sort this out add the inline suggestion and mark this as resolved.

> If you haven't already cloned the repository, do it and enter the repository:

Reviewer: ```suggestion
If you haven't already cloned the repository, do so and then enter the repository:
```

```console
git clone https://github.com/open-education-hub/operating-systems
cd operating-systems
```

Navigate to the chapter lab directory:

```console
cd content/chapters/software-stack/lab/

> ```

Reviewer: The Docker container defines some aliases to navigate more quickly [1]. I still think they're pointless, but maybe we should mention them here. @Alex-deVis would love this.

[1] https://github.com/open-education-hub/operating-systems/blob/b6a0cbdff828bdabfb7403a597e325c747e6bfbe/util/os-runner/config_terminal.sh#L21-L25

# Setting up the Lab Environment

If you have already cloned the repository, make sure it is updated:

```console

> cd operating-systems
> git pull --rebase

Reviewer: ```suggestion
student@os:~$ cd operating-systems

student@os:~/operating-systems$ git pull --rebase
```
It's better to leave an empty line between subsequent commands to make them more clearly distinguishable.

```

The command may fail if you have uncommitted changes.

> If that is the case, commit changes and retry.

Reviewer: ```suggestion
If that is the case, stash your changes, retry, and pop the stash:

```console
student@os:~/operating-systems$ git stash

student@os:~/operating-systems$ git pull --rebase

student@os:~/operating-systems$ git stash pop
```
```

The suggestion is a bit malformed due to the backticks interfering with each other.

If you haven't already cloned the repository, do so and then enter the repository:

```console

> git clone https://github.com/open-education-hub/operating-systems
> cd operating-systems

Reviewer: ```suggestion
student@os:~$ git clone https://github.com/open-education-hub/operating-systems

student@os:~$ cd operating-systems
```

```

Navigate to a chapter's lab directory:

```console

> cd content/chapters/<chapter-name>/lab/

Reviewer: ```suggestion
student@os:~/operating-systems$ cd content/chapters/<chapter-name>/lab/
```
